### ISSUE TITLE
Feature request: 'show-on-failure'

### ISSUE THREAD

**Issue by AdamHansrod** [toko-bifrost/ms-teams-deploy-card#10]
If you use this action as a step with an if conditional for job failure, then it won't run the steps necessary to show a card in a teams channel if the job fails. 

```
      - name: Notify channel
        uses: toko-bifrost/ms-teams-deploy-card@master
        with:
          github-token: ${{ secrets.GITHUB_TOKEN }}
          webhook-uri: ${{ secrets.TEAMS_WEBHOOK_URL }}
          show-on-start: false
          show-on-exit: true
          card-layout-start: compact
          card-layout-exit: compact
        if: ${{ failure() }}
```
Removing the if condition then notifies the channel for every completed job rather than only on job failures

It'd be great to have a 'show-on-failure' flag so that it only notifies when the job has failed rather than on any job exit

**Comment by aldwyn-acn** [comment]
@AdamHansrod, thanks for this suggestion. I will look into it.

**Comment by aldwyn-acn** [comment]
Hi @AdamHansrod. I just released [3.1.0](https://github.com/toko-bifrost/ms-teams-deploy-card/releases/tag/3.1.0) with this feature. Kindly check.
